fireblocks_sdk/sdk.py

```diff
@@ -113,18 +113,55 @@ def link_token(self, asset_id: str):
     def unlink_token(self, asset_id: str):
         return self._delete_request(f"/v1/tokenization/tokens/{asset_id}")
 
+    def get_staking_chains(self):
+        """Get all staking chains."""
+        return self._get_request("/v1/staking/chains")
+
+    def get_staking_chain_info(self, chain_descriptor: str):
+        """Get chain info."""
+        return self._get_request(f"/v1/staking/chains/{chain_descriptor}/chainInfo")
+
+    def get_staking_positions_summary(self):
+        """Get staking positions summary."""
+        return self._get_request(f"/v1/staking/positions/summary")
+
+    def get_staking_positions_summary_by_vault(self):
+        """Get staking positions summary by vault."""
+        return self._get_request("/v1/staking/positions/summary/vaults")
+
+    def execute_staking_action(self, chain_descriptor: str, action_id: str, request_body):
```

Comment:
Please add the relevant type to `request_body`:
`request_body: Union[StakeRequestDto, UnstakeRequestDto, WithdrawRequestDto]`